ExampleMod/Content/Projectiles/ExampleAdvancedAnimatedProjectile.cs

```diff
@@ -105,18 +106,18 @@ public class ExampleAdvancedAnimatedProjectile : ModProjectile
 				spriteEffects = SpriteEffects.FlipHorizontally;
 
 			// Getting texture of projectile
-			Texture2D texture = (Texture2D)ModContent.Request<Texture2D>(Texture);
+			Texture2D texture = TextureAssets.Projectile[Type].Value;
```

Comment:
Hm, it seems that modded entries in `TextureAssets.Projectile/Npc/Item` will always be loaded, where vanilla entries will be loaded on-demand via `Main.LoadProjectile/Item/NPC`.

I'm not sure what we should do about this, but it's a nuance that seems like a gotcha.